### ISSUE TITLE
Make app independent from Google services

### ISSUE THREAD

**Issue by megfault** [tjanson/WifiLocationLogger#21]
According to the [Freifunk forum discussion](https://forum.freifunk.net/t/reichweite-der-ff-knoten-in-der-map-darstellen/141/181), the app requires the Play Services API "FusedLocation", but in principle that could be changed.

Not everybody uses Google services, so if this is not really needed, can you please remove it?

**Comment by tjanson** [comment]
It’s possible to [use the Android location API directly](https://developer.android.com/guide/topics/location/strategies.html), and being independent of Google Play services is definitely a good goal. However, the docs also state:

> **This API is not the recommended method for accessing Android location.**
> The Google Location Services API, part of Google Play services, is the preferred way to add location-awareness to your app. It offers a simpler API, higher accuracy, low-power geofencing, and more. If you are currently using the android.location API, you are strongly encouraged to switch to the Google Location Services API as soon as possible.

So it’s certainly possible, but it might introduce new problems. Also, there are currently a ton of other issues that need attention more urgently (e.g., rotation bug, proper background service, UI & functionality improvements, DB backend & API, …).
In other words, this is at the very bottom of a long to-do list. Any help is welcome.

**Comment by megfault** [comment]
Sure, I understand there are more urgent things to do. Thanks for keeping the issue on the list, even if at the very bottom.

I just listed it here because I am one of those stubborn people that sticks to stock Android and doesn't want to use Google Services, but I guess we are a minority...

**Comment by alexanderadam** [comment]
Besides from that, removing proprietary dependencies (and adding a proper OS license file) would allow to publish the app on [the official F-Droid repositories](https://f-droid.org/) as well.

Even if the project seems to be abandoned in the last three years: thank you for the work! 👍

**Comment by IzzySoft** [comment]
Any news on this, @tjanson? Also, what license is your app using (that must be clear when applying for F-Droid inclusion)? Do you still maintain this app at all (2015 is quite a while ago, when the last commit was made)?

**Comment by IzzySoft** [comment]
@tjanson any answer on this? Is the app still maintained at all – or might it be better to "archive" the repo (which still allows forking for parties interested in picking up)?

**Comment by tjanson** [comment]
@IzzySoft oh, it's definitely not maintained, I haven't used it in years, let alone made changes.
